code/game/machinery/computer/camera.dm

```diff
@@ -245,6 +245,16 @@
 	light_color = "#3848B3"
 	light_power_on = 0.5
 
+obj/machinery/computer/security/wooden_tv/public
+	name = "public camera monitor"
+	desc = "An old TV hooked into the station's public cameras."
+	circuit = /obj/item/circuitboard/security/wooden_tv/public
+
+/obj/machinery/computer/security/wooden_tv/public/Initialize()
+	if(!network)
+		network = public_networks.Copy()
```

Comment:
why do we need a global list for this? just have the networks get copied as a direct list when it's made

Reply:
I tried doing that, but it didn't work.  I'll try again though, maybe I need to try harder/do it a different way.

Reply:
(I did it that way because the engineering camera monitor did it too.)

Reply:
Fixed!  Dunno what I did wrong.